### PR TITLE
VZ-9546: Rename component capi to clusterAPI

### PR DESCRIPTION
**Pull request by prasad-shirodkar**
Rename component capi to clusterAPI.

### CODE REVIEW

**Review by anders-swanson — DISMISSED** [review]

File: platform-operator/controllers/verrazzano/component/clusterapi/clusterapi_component.go [hunk 1]
```diff
@@ -24,13 +24,13 @@ import (
...
 // ComponentName is the name of the component
-const ComponentName = "capi"
+const ComponentName = "clusterAPI"
 
 // Namespace for CAPI providers
 const ComponentNamespace = constants.VerrazzanoCAPINamespace
 
 // ComponentJSONName is the JSON name of the component in CRD
-const ComponentJSONName = "capi"
+const ComponentJSONName = "clusterAPI"
```
Comment (bobdonat) [inline platform-operator/controllers/verrazzano/component/clusterapi/clusterapi_component.go:33]: I think this is the name that will show up on the vz resource component status.  Is that the name we want?

Comment (prasad-shirodkar) [reply platform-operator/controllers/verrazzano/component/clusterapi/clusterapi_component.go:33]: yes that's exactly we want now.

Comment (prasad-shirodkar) [reply platform-operator/controllers/verrazzano/component/clusterapi/clusterapi_component.go:33]: or should it be something like `cluster-api` keeping in line with other names?

Comment (bobdonat) [reply platform-operator/controllers/verrazzano/component/clusterapi/clusterapi_component.go:33]: not sure, a similar question came up last week for another component. @kminder what do you think?

Comment (kminder) [reply platform-operator/controllers/verrazzano/component/clusterapi/clusterapi_component.go:33]: Unfortunately in spec it needs to be clusterAPI and in status needs to be cluster-api.  I'm not sure what needs to change in the code to accomplish this.

**Review by bobdonat — DISMISSED** [review]

**Review by bobdonat — APPROVED** [review]

**Pull request merged**